scripts/src/lib/template/templates/gradle/kotlin/build.gradle.kts.eta

```diff
@@ -0,0 +1,113 @@
+plugins {
+	id("fabric-loom") version "1.0-SNAPSHOT"
+	id("maven-publish")<% if (it.kotlin) { %>
+	kotlin("jvm") version "<%= it.kotlin.kotlinVersion %>"<% } %>
+}
+
+java.sourceCompatibility = JavaVersion.<%= it.java.compatibility %>
+java.targetCompatibility = JavaVersion.<%= it.java.compatibility %>
+
+base.archivesName.set(project.properties["archives_base_name"] as String)
+version = project.properties["mod_version"] as String
+group = project.properties["maven_group"] as String
```

Comment:
```suggestion
base.archivesName.set(project.property("archives_base_name").toString())
version = project.property("mod_version").toString()
group = project.property("maven_group").toString()
```
iirc computing the full `properties` map is pretty slow + it's longer in code, and `toString` is safer than a cast

Reply:
toString is safer than as String, but when the property isn't a string, then it shouldn't work anyway...  
I thing as String (in this case) is better.  

property(...) idk.. The map is only calculated once (the performance impact is negligible), and using a map makes more sense than a function. At least for me.

Reply:
The function looks cleaner imo, and it's shorter. The cast/toString can go either way though. (And I don't think anything else computes the map anyway so it is slower in that case)